cmd/internal/register/maps.go

```diff
@@ -217,12 +211,12 @@ func Maps(a *atlas.Atlas, confMaps []config.Map, providers map[string]provider.T
 				}
 			}
 
-			var minZoom uint
+			minZoom := uint(1)
 			if l.MinZoom != nil {
 				minZoom = uint(*l.MinZoom)
 			}
 
-			var maxZoom uint
+			maxZoom := uint(20)
```

Comment:
We have a const for this [here](https://github.com/go-spatial/tegola/blob/573d26150c34970fcd54afabfbbc64f4a347609b/atlas/atlas.go#L21)